### DISCUSSION TITLE
Cidre in the wild

### DISCUSSION THREAD

**Discussion by morajabi** [yury/cidre#11] [General]
I think I got you a couple new stars on the repo :)

https://x.com/morajabi/status/1754427380054053308?s=20

**Comment by yury** [comment]
Thank you, sir!

**Reply by morajabi** [reply]
One more mention of Cidre!
https://crabnebula.dev/blog/interview-with-mo/

![CleanShot 2024-04-19 at 13 23 12](https://github.com/yury/cidre/assets/12202757/c0abe810-1288-4bb7-9a97-eddce7c893ce)

**Reply by yury** [reply]
Thank you, sir!

**Comment by morajabi** [comment]
One more time! https://x.com/dceddia/status/1887984368951709758

**Reply by yury** [reply]
thank you again!
